src/types/FullTodo.ts

```diff
@@ -0,0 +1,6 @@
+import { Todo } from './Todo';
```

Comment:
Consider using relative path instead of absolute path for better maintainability. 🛣️